examples/mxnet_imagenet_resnet50.py

```diff
@@ -16,25 +16,25 @@
 # SOFTWARE.
 
 import argparse
-import time
 import logging
+import math
 import os
-import numpy as np
+
+
+from gluoncv.model_zoo import get_model
+import horovod.mxnet as hvd
 import mxnet as mx
-import math
-from mxnet import gluon, nd
-from mxnet import autograd as ag
-from mxnet.lr_scheduler import MultiFactorScheduler
+import numpy as np
+from mxnet import gluon
+from mxnet import lr_scheduler
 from mxnet.io import DataBatch, DataIter
-from mxnet.util import makedirs
-from mxnet.gluon.model_zoo.vision import get_model
-import horovod.mxnet as hvd
+
 
 # Training settings
 parser = argparse.ArgumentParser(description='MXNet ImageNet Example',
                                  formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-parser.add_argument('--benchmark', type=int, default='0',
```

Comment:
This argument is actually being used in imagenet example in MXNet: https://github.com/apache/incubator-mxnet/blob/ceabcaac77543d99246415b2fb2d8c973a830453/example/image-classification/common/data.py

Reply:
I was following gluoncv example. I replace this one with use-rec which I think does the same thing: controlling if to use synthetic data.